### PR TITLE
Example program to export systems to a CSV file

### PR DESCRIPTION
**Pull request by jamesbrown7**

### CODE REVIEW

**Comment by kusold** [comment]
:+1:

File: examples/ExportSystemsToCSV/exportSystemsToCSV.go [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+const (
+	apiUrl string = "https://console.jumpcloud.com/api"
+)
+
+func outFirst(data string) {
+	fmt.Printf("\"%s\"", data)
+}
+
+func out(data string) {
+	fmt.Printf(",\"%s\"", data)
```
Comment (bryancallahan) [inline examples/ExportSystemsToCSV/exportSystemsToCSV.go]: Do we need to do any `"` escaping in the `data` for these guys?

**Comment by bryancallahan** [comment]
:+1: Looks good (one tiny comment if it's a concern)

**Comment by bryancallahan** [comment]
:+1:

**Comment by kusold** [comment]
:+1:

**Pull request merged**